tubular-react-common/index.md

```diff
@@ -33,13 +33,54 @@ description: "Check how to improve your CRUD views with amazing webcontrols from
           Function - That adds another count whenever grid is refreshed.
         </p>
       </div>
-      <div>
-        <iframe
-          src="https://codesandbox.io/embed/uno-reactuseclickoutside-demo-flf7c?fontsize=14&hidenavigation=1&theme=dark&previewwindow=console&view=split"
-          style="width:100%; height:300px; border:0; border-radius: 4px; overflow:hidden;"
-          title="useClickOutside" sandbox="allow-scripts allow-same-origin"></iframe>
-      </div>
     </div>
+```tsx
+{% raw %}
+import React from "react";
+import "./styles.css";
```

Comment:
Keep it simple!

Reply:
I believe I did. I editted the code on the examples